tools/WinObjC.Packaging/Package.Consumption.targets

```diff
@@ -0,0 +1,64 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+
+<!--
+  PACKAGE LINKER TARGETS
+  Description: Linking of default package internals
+  For Use By: A VcxProj that consumes this package
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Categorize files from the package -->
+  <ItemGroup Condition="'$(ConfigurationType)' != 'StaticLibrary'">
+    <_DllsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\*.dll"/>    
+    <_DllsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.dll"/>
+      
+    <_ReferencesFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\*.winmd"/>    
+    <_ReferencesFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.winmd"/>
+
+    <_ImportLibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\*.lib"/>    
+    <_ImportLibsFromPackage Include="$(MSBuildThisFileDirectory)\lib\$(TargetOsAndVersion)\$(PlatformTarget)\$(Configuration)\*.lib"/>
+
+    <_XamlFilesFromPackage Include="$(MSBuildThisFileDirectory)\xaml\**\*.xaml" />
+    <_BundleFilesFromPackage Include="$(MSBuildThisFileDirectory)\bundle\**\*.*" />
+    <_ContentFilesFromPackage Include="$(MSBuildThisFileDirectory)\content\**\*.*" />
+  </ItemGroup>
+
+  <!-- Add references to package files for projects that consume this package -->
+  <ItemGroup Condition="'$(ConfigurationType)' != 'StaticLibrary'"> 
```

Comment:
>ItemGroup Condition="'$(ConfigurationType)' != 'StaticLibrary' [](start = 3, length = 62)

was thinking and we might actually want to remove this so that the import lib is bundled in a static lib too. The scenario I encountered that made me think of this was if we had a dll project that is straight c++ and just has a project reference to a static lib that is objC. The dll would need to also need to depend on the nuget to get it to link sucessfully but thats a little weird. 


So I think what we want to do is always link but only do the ReferenceCopyLocalPaths if its not a static lib.